src/manual/task-execution/provisioning.md

```diff
@@ -0,0 +1,46 @@
+---
+filename: task-execution/provisioning.md
+title: Provisioning
+order: 40
+---
+
+In the simplest case, a worker implementation is installed and configured
+manually on a host and executes tasks indefinitely. For most purposes, it is
+far more cost-effective to dynamically adjust the pool of available workers
+based on the work to be done. This process is called provisioning.
+
+A provisioner operates by monitoring the pool of running workers and the
+corresponding queues. Based on the queue length and the configuration of the
+provisioner, it may create new workers or signal to running workers that they
+should stop. Of course, provisioners might take into account additional
```

Comment:
So the original design for the general provisioner contract was that it absolutely would not signal a shutdown.  This is still 100% true, the 96 hour shutdown is a safety feature and does no signalling to the worker, rather it will just hard-shutdown the machine after 96 hours.

The specific details of the aws-provisioner contract should go in that section: all worker types are expected to shutdown at most 72 hours after they boot and will be forcefully killed after 96 without warning.